dbt_project.yml

```diff
@@ -57,7 +57,7 @@ vars:
     netsuite2_transactions: "{{ ref('stg_netsuite2__transactions') }}"
     netsuite2_vendor_categories: "{{ ref('stg_netsuite2__vendor_categories') }}"
     netsuite2_vendors: "{{ ref('stg_netsuite2__vendors') }}"
-    accounts_pass_through_columns: []
+    accounts_pass_through_columns: [sspecacct]
```

Comment:
I have renamed it `sspecacct as special_account_type_id,` at upstream, per the naming conventions

Reply:
Sorry, I'm not following.

Reply:
https://github.com/rwang-lyra/dbt_netsuite_source_fork/blob/main/models/netsuite2/stg_netsuite2__accounts.sql#L30
here

Reply:
Got it. I see the renamed field now in `STG_NETSUITE2__ACCOUNTS`.